from connection import get_db_connection

try:
    db = get_db_connection()
    
    db.ping(reconnect=True) 
    print("Conex√£o bem-sucedida!")
    db.close()
except Exception as e:
    print(f"Erro: {e}")
